include/tenstorrent/status_reg.h

```diff
@@ -4,11 +4,13 @@
  */
 
 /* Scratch registers used for status and error reporting */
-#ifndef STATUS_REG_H
-#define STATUS_REG_H
+#ifndef TESNTORRENT_STATUS_REG_H_
+#define TENSTORRENT_STATUS_REG_H_
 
 #include <stdint.h>
 
+#define RESET_UNIT_REFCLK_CNT_LO_REG_ADDR 0x800300E0
```

Comment:
This isn't exactly a status register. It belongs to the timer, so by moving this here we are leaking between subsystems or drivers and adding tech debt.